examples/Dropdown.elm

```diff
@@ -81,21 +81,21 @@ noDate whenText =
 
 
 formatDate : Date -> String
-formatDate date =
-    (toString (Date.day date)) ++ " " ++ (toString (Date.month date)) ++ " " ++ (toString (Date.year date))
+formatDate =
+    Date.format "d MMM y"
```

Comment:
this is much tidier and hopefully easier for people to understand.
we would obviously prefer to not add dependencies, but it should be fine. 👍